scheduler/src/cook/mesos/util.clj

```diff
@@ -717,7 +717,7 @@
         schedule (tp/periodic-seq start-time frequency)
         chime-fn (fn [_]
```

Comment:
We should have an error handler here so we can have proper logging.

Reply:
Done.